src/components/article/CreateArticle.jsx

```diff
@@ -74,7 +74,7 @@ CreateArticle.propTypes = {
 };
 
 const mapStateToProps = ({ auth, article }) => ({
-  isLoggedIn: auth && auth.isLoggedIn ? auth.isLoggedIn : null,
```

Comment:
Yeah @okoroemeka, it occurred to me there's no need doing so many checks just to map a state from our store to the `isLoggedIn` props since we already set `initialState` for the auth state as 

```javascript
const initialState = { user: null, isLoggedIn: null };
```

the line below is checking if there's a state called `auth` in our store and checking that a property (`isLoggedIn`) is true else return null.

```javascript 
auth && auth.isLoggedIn ? auth.isLoggedIn : null,
```

This also introduces a bug for when our app first mount and we try to get the logged in user (asynchronously) but this particular component renders after which our async action has not finish. We are already saying in `mapStateToProps` that "when the `isLoggedIn` is not true, return null". i.e. even before our async action for getting the current user is done, we are setting the `isLoggedIn` state for that user null.